frontend/src/metabase/admin/people/containers/AdminPeopleApp.jsx

```diff
@@ -3,28 +3,50 @@ import { Component } from "react";
 import PropTypes from "prop-types";
 import { t } from "ttag";
 
+import _ from "underscore";
+import { connect } from "react-redux";
 import { LeftNavPane, LeftNavPaneItem } from "metabase/components/LeftNavPane";
+import NudgeToPro from "metabase/admin/people/components/NudgeToPro";
 
 import AdminLayout from "metabase/components/AdminLayout";
+import { shouldNudgeToPro } from "metabase/admin/people/selectors";
 
-export default class AdminPeopleApp extends Component {
+class AdminPeopleApp extends Component {
   static propTypes = {
     children: PropTypes.any,
+    shouldNudge: PropTypes.bool,
   };
 
   render() {
-    const { children } = this.props;
+    const { children, shouldNudge } = this.props;
+    const sidebar = (
+      <LeftNavPane fullHeight={!shouldNudge}>
+        <LeftNavPaneItem name={t`People`} path="/admin/people" index />
+        <LeftNavPaneItem name={t`Groups`} path="/admin/people/groups" />
+      </LeftNavPane>
+    );
     return (
       <AdminLayout
         sidebar={
-          <LeftNavPane>
-            <LeftNavPaneItem name={t`People`} path="/admin/people" index />
-            <LeftNavPaneItem name={t`Groups`} path="/admin/people/groups" />
-          </LeftNavPane>
+          !shouldNudge ? (
+            sidebar
+          ) : (
+            <div className="flex flex-column">
+              {sidebar}
+              <NudgeToPro />
+            </div>
+          )
         }
       >
         {children}
       </AdminLayout>
     );
   }
 }
+
+// selectors:
```

Comment:
Nit: can probably remove the comment since `mapStateToProps` is ubiquitous throughout the codebase.